### PR TITLE
Fix static analysis errors

### PR DESCRIPTION
**Pull request by pavlakis**

### CODE REVIEW

File: app/src/Model/Event/Entity/Event.php [hunk 1]
```diff
@@ -4,22 +4,49 @@
...
+     * @var int
+     */
     private $supporterID;
 
+    /**
+     * @var \DateTime
+     */
     private $meetupDate;
 
     public function __construct($meetupID, $meetupVenueID, $joindinEventName, $joindinTalkID, $joindinURL, $speakerID, $supporterID, \DateTime $meetupDate)
```
Comment (sdh100shaun) [inline app/src/Model/Event/Entity/Event.php]: can any of these be type hinted?

File: app/src/Model/Event/EventManager.php [hunk 2]
```diff
@@ -93,19 +93,19 @@ public function saveEvent(EventEntity $event)
...
     }
 
     /**
-     * @param $meetupID
-     * @return bool
+     * @param int $meetupID
+     * @return array
      */
-    public function getByMeetupID($meetupID)
+    public function getByMeetupID(int $meetupID)
```
Comment (sdh100shaun) [inline app/src/Model/Event/EventManager.php]: return type

File: app/src/Model/Event/EventManager.php [hunk 3]
```diff
@@ -128,7 +128,12 @@ public function getAllPendingEvents()
...
 
-    public function getByYearMonth($year,$month)
+    /**
+     * @param int $year
+     * @param int $month
+     * @return array
+     */
+    public function getByYearMonth(int $year, int $month)
     {
         $event = $this->eventsRepo->getEventByYearAndMonth($year,$month);
```
Comment (sdh100shaun) [inline app/src/Model/Event/EventManager.php]: return type

File: app/src/Model/Event/EventManager.php [hunk 4]
```diff
@@ -59,25 +59,25 @@ public function getSupporters()
     }
 
     /**
-     * @param $supporterID
-     * @return Entity\Supporter
+     * @param int $supporterID
+     * @return SupporterInterface
      */
-    public function getSupporterByID($supporterID)
+    public function getSupporterByID(int $supporterID)
```
Comment (sdh100shaun) [inline app/src/Model/Event/EventManager.php]: return type

File: app/src/Model/MeetupEvent.php [hunk 5]
```diff
@@ -193,21 +193,21 @@ public function getMeetupEventID()
 
         $id = substr($this->getEventLocation(), strlen($this->baseUrl . '/event/'));
         if (substr($id, -1) == '/') {
```
Comment (sdh100shaun) [inline app/src/Model/MeetupEvent.php]: should we use === here ?

**Comment by pavlakis** [comment]
Closing as we need more tests for regression before attempting to fix static analysis errors.

**Pull request closed without merging**